scalyr_agent/builtin_monitors/windows_event_log_monitor.py

```diff
@@ -294,8 +302,14 @@ def GetFormattedEventAsDict( self, render_context, event ):
 
         result = {}
 
-        event_id = vals[win32evtlog.EvtSystemEventID]
-        qualifiers = vals[win32evtlog.EvtSystemQualifiers]
+        # build the EventId
+        event_id_val = vals[win32evtlog.EvtSystemEventID]
+        if event_id_val[1] != win32evtlog.EvtVarTypeNull:
+            event_id = event_id_val[0]
+            qualifiers_val = vals[win32evtlog.EvtSystemQualifiers]
+            if qualifiers_val[1] != win32evtlog.EvtVarTypeNull:
+                event_id = win32api.MAKELONG( event_id, qualifiers_val[0] )
```

Comment:
Can you add some comments explaining what is going on here?

Reply:
Done.  If you'd prefer, we could also split this in to 3 separate fields:

1. EventID
2. SystemQualifiers
3. InstanceID

(see comments in the new commit to see how these are related).

Reply:
If we strongly thought folks would want to search on those independently, then it would make sense. I don't know if we have strong data one way or the other, so let's just keep it simple for now.